src/main/java/com/scalyr/integrations/kafka/AddEventsClient.java

```diff
@@ -74,10 +73,10 @@
   // Separate logger which is used for logging the whole payload when the payload exceeds the limit
   private static final Logger payloadLogger = LoggerFactory.getLogger("com.scalyr.integrations.kafka.eventpayload");
 
-  private final CloseableHttpClient client = HttpClients.createDefault();
+  private static final AsyncHttpClient asyncHttpClient = asyncHttpClient();
   private final ObjectMapper objectMapper = new ObjectMapper();
-  private final ExecutorService senderThread = Executors.newSingleThreadExecutor();
-  private final HttpPost httpPost;
+  private final ExecutorService retryThread = Executors.newSingleThreadExecutor();
+  private final BoundRequestBuilder requestBuilder;
```

Comment:
Can you add documentation for this variable?  The bound thing raises an eyebrow for me.